metsrw/premis_event.py

```diff
@@ -0,0 +1,40 @@
+from . import exceptions
+from . import utils
+
+
+class Event(object):
```

Comment:
Rename to PremisEvent (or PREMISEvent) to distinguish from some other metadata's events?  Or maybe put all the PREMS things in premis.py so you have premis.Event?